src/routes/route-helpers.ts

```diff
@@ -1,7 +1,19 @@
 import { generatePath } from "react-router-dom";
-import { ParamsHelper } from "helpers/params-helper";
-import { routes, RouteName } from "./routes";
-import type { Params } from "./routes";
+import { routes } from "./routes";
+import type { Params, RouteName } from "./routes";
+
+const buildQueryParams = (params: Params) => {
+  const entries = Object.entries(params);
+  if (!entries.length) {
+    return "";
+  }
+  const queryParams = new URLSearchParams();
+  for (const [key, value] of entries) {
+    queryParams.set(key, String(value));
+  }
+  const stringParams = queryParams.toString();
+  return `${stringParams.length ? "?" : ""}${stringParams}`;
```

Comment:
```suggestion
  const queryString = new URLSearchParams(params).toString();
  return queryString.length ? `?${queryString}' : '';
```

Reply:
This is actually a very good catch! Thanks!